### DISCUSSION TITLE
don't limit raid tradeable transmog

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3537] [Suggestions]
Edit: so staff have said it's the module creator changing it rather than them. I still think it's worth keeping this up so they know we don't like it and hopefully they can find an alternative.

On the latest change in #Changelog-chromie, I think transmog collecting should remain as it works now. 

Raid gear is very limited and takes effort to obtain. If a good looking piece of loot drops, everyone in the group has earned it equally. Transmog is a low priority when distributing loot, so players who want to collect appearances will have a much harder time.

Further limiting transmog only reduces player fun. 
I don't see any reason why this change would be a positive.

Edit: at least give us a way to soulbind easily (other than enchanting) so that i can sell gear without having to wait like 2 hours. doesnt even tick down when offline.

**Comment by Slysam** [comment]
I think that if the Transmog system had worked this way from the beginning(as in, worked the way they're changing it to work), then it wouldn't necessarily be an issue. However, it hasn't worked this way.

The problem becomes that players have already gotten used to the way the system works, raids don't have to bicker and fight over people wanting/rolling on loot that someone else actually needs just so they can feed their vanity, and ninja looting pretty swords or armor for appearances doesn't happen.

Changing Transmog after it's already existed in this state really just feels like a step backward, and does nothing but alter a system that players are very happy with in the name of 'blizzlike', when it was actually a very positive change. This seems like a decision that is ONLY going to be incredibly unpopular, and will re-introduce very unhealthy and frustrating behavior that the current system entirely avoids.

I see no real benefit to this change and an incredible number of downsides, and I fully support either reversing this change or simply not adding it at all to begin with, at least not without a significant amount more discussion with the community about why and allowing them to weigh in on it.

Edit:
It was pointed out that this update to the Transmog system is coming because it was made by the person who made the mod originally, and was not something "intentionally" added by the CC staff. This does change how I feel about the change(after all, it wasn't something that they intended to do) but it doesn't change that I think that this is very detrimental for very little benefit. If it is possible to update or otherwise "fix" this change, I think it should be implemented as quickly as possible, and I stand by my assessment of the downsides of the change.

**Comment by plasma-93** [comment]
![image](https://user-images.githubusercontent.com/38546519/170343511-e5e2b1f8-3b75-4be8-a6a0-215056588e68.png)
Just to not have to retype my points again, a screenshot of the discord.

**Comment by Spook9** [comment]
This adds potential problems in the form of loot drama and no upsides. Why would you change a working system?

**Comment by drainedaugustus** [comment]
Purely negative change that only serves to take away something a lot of people were enjoying with no actual material impact on gameplay

**Comment by Hihilist** [comment]
To add insult to injury - with current change getting any equipment that can be traded does not add it to collection when they are equipped. You need to literally wait 2 hrs to equip it, so it is in your collection, and then you can get rid of it (sell/delete).

That is not friendly UX, as it uses inventory space and requires to remember which gear was already added to collection and which not yet.

**Reply by BeefSteaker** [reply]
Workaround is getting cheap enchants from power levelling enchanting, as when an item is enchanted it removes the trade window

**Reply by Hihilist** [reply]
It is an option, yes...BUT...sounds like a band aid, not a solution. Plus still proves it is bad UX - let me invest more into piece that I grabbed only for its look. The cheapest enchants on AH start from 5g and most collection pieces will not cover that.
Buying separate mats and looking for enchanter also is not good UX. Plus not all ppl power levelled enchanting and definitely will not do that to remove trade window just for collection purposes.

**Comment by heyitsbench** [comment]
Items in raids should soon be able to be traded for transmog once this commit makes its way to CC.

https://github.com/azerothcore/mod-transmog/pull/86

**Reply by BeefSteaker** [reply]
Thank you Pepsi man

**Comment by Nyeriah** [comment]
Thanks @benjymansy123 for the PR. We will have it enabled in the next update.

Locking this one as it was accepted.

Regards.
